pkg/client/didexchange/client_test.go

```diff
@@ -474,19 +475,24 @@ func TestServiceEvents(t *testing.T) {
 	newDidDoc, err := (&mockcreator.MockDIDCreator{}).Create("test")
 	require.NoError(t, err)
 
+	didService, err := service.GetPthid(newDidDoc.Service)
```

Comment:
Is there a need to expose GetPthid at service level? This looks more like private utility function.